packages/functional/actions/cart-actions.ts

```diff
@@ -0,0 +1,26 @@
+import { Page } from '@playwright/test';
+
+export async function addEstimatedShippingCosts(
+  page: Page,
+  country: string,
+  state: string,
+  city: string,
+  zip: string,
+): Promise<void> {
+  await page.getByRole('combobox', { name: 'Country' }).click();
+  await page.getByLabel(country).click();
+
+  await page
+    .getByRole('combobox', { name: 'State' })
+    .filter({ hasNotText: 'Select state' })
```

Comment:
I think we should be able to change it to this in order to not use `.filter`.
```suggestion
    .getByRole('combobox', { name: 'State/province' })
```

Reply:
This was done mainly to avoid sporadic failures which were happening when playwright was quick to click on the dropdown before it loads the states based on selection of country. This was happening when testing against localhost and so, I tried to handle it for all scenarios like running it on CI and locally. I'll try and verify if it's working fine without the filter.